warehouse/accounts/views.py

```diff
@@ -169,3 +169,42 @@ def logout(request, redirect_field_name=REDIRECT_FIELD_NAME):
         return HTTPSeeOther(redirect_to, headers=dict(headers))
 
     return {"redirect": {"field": REDIRECT_FIELD_NAME, "data": redirect_to}}
+
+
+@view_config(
+    route_name="accounts.register",
+    renderer="accounts/register.html",
+    decorator=[csrf_protect("accounts.register"), uses_session],
+)
+def register(request, _form_class=forms.RegistrationForm):
+    if request.authenticated_userid is not None:
+        return HTTPSeeOther("/")
```

Comment:
I guess this raises the question of what exactly should a /register/ page *do* if there is a logged in user... I suspect redirecting to something is a reasonable thing though. This should use ``request.route_path("index")`` though to avoid hardcoding the path.